js/pricing.js

```diff
@@ -120,33 +135,43 @@ $(
          * @typedef {Object} Consent
          * @property {boolean} privacyConsent indicates whether the user gives consent to
          * process his personal information
-         * @property {boolean} supportAgreementConsent indicates whether the user agrees to be bound by the
-         * terms of the Development Support Agreement
+         * @property {boolean} supportAgreementConsent indicates whether the user agrees to be
+         * bound by the terms of the Development Support Agreement
+         */
+
+        /**
+         * The user's consent transaction data.
+         *
+         * <p>The user's consents and the reCaptcha token. The token uses to verify whether it is a
+         * real user or some malicious code.
+         *
+         * @typedef {Object} ConsentTransaction
```

Comment:
It is not a transaction, right?

The transaction only will be created if the creation request is successful.